gormtestsupport/db_test_suite.go

```diff
@@ -165,3 +170,24 @@ func (s *DBTestSuite) WrapFactory(identifier string, constructor wrapper.Factory
 func (s *DBTestSuite) ResetFactories() {
 	s.Wrappers.ResetWrappers()
 }
+
+// Run overrides the default behaviour of the Suite.Run method, in order
+// to run the SetupSubtest and TearDownSubtest methods for each subtest
+func (s *DBTestSuite) Run(name string, subtest func()) bool {
+	fmt.Printf("==== RUN Subtest '%s'\n", name)
+	if s.SetupSubtest != nil {
+		s.SetupSubtest()
+	}
+	defer func() {
+		fmt.Printf("==== END Subtest '%s'\n", name)
+		if s.TearDownSubtest != nil {
+			s.TearDownSubtest()
+		}
+	}()
+	return s.Suite.Run(name, subtest)
+}
+
+// Skip keeps the code for the test, but skips its execution
+func (s *DBTestSuite) Skip(name string, subtest func()) bool {
+	return true
```

Comment:
Why?

Reply:
removed in 239b06e: this function is not used